src/cockpit/bridge.py

```diff
@@ -114,7 +114,7 @@ def get_os_release():
             return dict(token.split('=', 1) for token in lexer)
 
     def do_init(self, message: Dict[str, object]) -> None:
-        superuser = message.get('superuser')
+        superuser = message.get('superuser', {})
         if isinstance(superuser, dict):
```

Comment:
If you go with this change, then the `if` is always True, so you can drop it.

Reply:
No, the message might contain `"superuser": false`, for example.